settings.gradle.kts

```diff
@@ -8,5 +8,8 @@ gradleEnterprise {
     buildScan {
         termsOfServiceUrl = "https://gradle.com/terms-of-service"
         termsOfServiceAgree = "yes"
+        if (!System.getenv("CI").isNullOrEmpty()) {
```

Comment:
@gnarea not sure if this has a "cost", but it saved me a lot of time, so I kept it. Let me know if I should remove it.

Reply:
No, this is good, let's keep it. All repos should have this eventually.